README.rst

```diff
@@ -1 +1,15 @@
-Please refer to src/zope/globalrequest/README.rst.
+zope.globalrequest
+==================
+
+Introduction
+------------
+
+This package provides a global way to retrieve the currently active request object in a zope-based web framework.
```

Comment:
(I've just realized that you merely moved the text from the other README.rst here.)